huaweicloud-sdk-python-modelarts-dataset/modelarts/test/test_manifest_classification_detection_get_sample_list.py

```diff
@@ -15,21 +15,34 @@
 import os
 import sys
 
-import field_name
-import manifest
+from modelarts import manifest, field_name
 
 
 def check_data(sample_list):
+  assert sample_list.__len__() == 5
+  for raw_data, label_list in sample_list:
+    assert str(raw_data).startswith("s3://obs-ma/test/classification/datafiles")
+    assert label_list.__len__() == 1 or label_list.__len__() == 0
+
+
+def check_data_usage_inference(sample_list):
+  assert sample_list.__len__() == 3
+  for raw_data, label_list in sample_list:
+    assert str(raw_data).startswith("s3://obs-ma/test/classification/datafiles")
+    assert label_list.__len__() == 0
```

Comment:
why not use len(label_list) instead of label_list.__len__()

Reply:
What benefit for using len(label_list) instead of label_list.__len__()?